test/Tests.Infrastructure/RavenTestBase.Backup.cs

```diff
@@ -227,7 +228,11 @@ public async Task RunBackupInClusterAsync(DocumentStore store, long taskId, bool
 
             public IDisposable RestoreDatabase(IDocumentStore store, RestoreBackupConfiguration config, TimeSpan? timeout = null)
             {
-                var restoreOperation = new RestoreBackupOperation(config);
+                RestoreBackupOperation restoreOperation;
+                if (config.NodeTag != null)
+                    restoreOperation = new RestoreBackupOperation(config, config.NodeTag);
+                else
+                    restoreOperation = new RestoreBackupOperation(config);
```

Comment:
it is okay to have this, but why is that required for the test?

Reply:
I want to restore it to a specific node.